### PR TITLE
fix several typos in pkg/controller/nodeipam, nodelifecycle

### PR DESCRIPTION
**Pull request by chymy**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fix several typos in `pkg/controller/nodeipam, nodelifecycle`
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go [hunk 1]
```diff
@@ -1480,7 +1480,7 @@ func (nc *Controller) markNodeAsReachable(ctx context.Context, node *v1.Node) (b
 // ComputeZoneState returns a slice of NodeReadyConditions for all Nodes in a given zone.
 // The zone is considered:
 // - fullyDisrupted if there're no Ready Nodes,
-// - partiallyDisrupted if at least than nc.unhealthyZoneThreshold percent of Nodes are not Ready,
+// - partiallyDisrupted if at less than nc.unhealthyZoneThreshold percent of Nodes are not Ready,
```
Comment (yangjunmyfm192085) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go:1483]: Is it `at least` ?

Comment (yangjunmyfm192085) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go:1483]: `notReadyNodes > 2 && float32(notReadyNodes)/float32(notReadyNodes+readyNodes) >= nc.unhealthyZoneThreshold:`

Comment (chymy) [reply pkg/controller/nodelifecycle/node_lifecycle_controller.go:1483]: Oh, right.

**Review by yangjunmyfm192085 — COMMENTED** [review]
/lgtm

**Review by yangjunmyfm192085 — COMMENTED** [review]
/assign @bowei

**Pull request closed without merging**